src/Umbraco.Core/Services/Querying/RecycleBin/IDocumentRecycleBinQueryService.cs

```diff
@@ -0,0 +1,8 @@
+using Umbraco.Cms.Core.Models.Entities;
+
+namespace Umbraco.Cms.Core.Services.Querying.RecycleBin;
+
+public interface IDocumentRecycleBinQueryService
+{
+    Task<Attempt<IDocumentEntitySlim?, RecycleBinQueryResultType>> GetOriginalParentAsync(Guid trasherDocumentId);
```

Comment:
`trasherDocumentId` should be aligned with the name from media `trashedMediaFolderId`

Reply:
typo, should be trashedDocumentId and trashedMediaId